paymentsheet-example/src/androidTestDebug/java/com/stripe/android/test/core/DisableAnimationsRule.kt

```diff
@@ -0,0 +1,36 @@
+package com.stripe.android.test.core
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class DisableAnimationsRule : TestRule {
```

Comment:
Can you add a comment to make clear that we need this because we can’t pass this information to BrowserStack in another way?